src/additional/greaterThan.js

```diff
@@ -0,0 +1,11 @@
+$.validator.addMethod( "greaterThan", function( value, element, param ) {
+    var target = $( param );
+
+    if ( this.settings.onfocusout && target.not( ".validate-greaterThan-blur" ).length ) {
```

Comment:
Why do we need this focus handling?

Reply:
My motivation was to follow the pattern of the [equalTo](https://github.com/jquery-validation/jquery-validation/blob/master/src/core.js#L1466) method, and this was introduced there two years ago in [Core: Bind the `blur` event just once in `equalTo` rule ](https://github.com/jquery-validation/jquery-validation/commit/ef223608a31e487001006ea03776024cf46622cd).

Reply:
Just a clarification, the focus handling was added +5 years ago in this commit https://github.com/jquery-validation/jquery-validation/commit/aca144b5eba1f1947b311819e9eea8e0f176f448#diff-a692042a498ec7e201f38855d169e7b1R1163. The only thing that was added in the commit linked by @RobJohnston is binding the blur event one time.